src/core/player.js

```diff
@@ -278,15 +278,18 @@ DM.provide('Player',
     _decodePostMessage: function(rawMessage)
     {
       if (rawMessage.substring(0, 1) === '{') {
+        // apimode = json
         try {
           var data = JSON.parse(rawMessage);
           return data;
         }
         catch(e) {
           return {};
         }
+      } else {
```

Comment:
Can't see the diff with the previous version. It's just a matter of style, right? ("symmetry" vs. "fallback"?)

Reply:
I think it's to see better the `apimode` json OR queryString

Reply:
I think it's to see better the apimode json OR queryString

Reply:
yep it's just for symmetry that i did it this way